arlas-opensearch/src/main/java/io/arlas/server/rest/explore/opensearch/OpenSearchDescriptorService.java

```diff
@@ -128,7 +134,7 @@ public Response opensearch(
             description.syndicationRight = os.syndicationRight;
             description.tags = os.tags;
         }
-        addURLs(prefix, description.url, new ElasticAdmin(exploreServices.getClient()).describeCollection(cr).properties, new Stack<>());
+        addURLs(prefix, description.url, admin.describeCollection(cr, columnFilter).properties, new Stack<>());
```

Comment:
We can use `exploreServices.getElasticAdmin()`
we avoid creating another instance of the admini and the creation of `admin` variable